autoware_adapi_v1_msgs/vehicle/msg/VehicleState.msg

```diff
@@ -0,0 +1,6 @@
+builtin_interfaces/Time stamp
+float64 steering_tire_angle
+autoware_adapi_v1_msgs/Gear gear
+autoware_adapi_v1_msgs/TurnIndicator turn_indicator
+autoware_adapi_v1_msgs/HazardLight hazard_light
+float32 energy_level
```

Comment:
```suggestion
float32 energy_level
```
Is this a battery percentage? Fuel level percentage? Could you add it as a comment documentation?

Reply:
Actually this can be battery percentage or fuel percentage, it will depend on the vehicle specific.
I will add a comment documentation

Reply:
i added the commend document in this [c34bfed](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/24/commits/c34bfedc3e913492e7fdfd76eaacf2152de59719)